tutorials/preprocessing/plot_70_fnirs_processing.py

```diff
@@ -36,15 +36,17 @@
 # Here we validate that the location of sources-detector pairs and channels
 # are in the expected locations. Source-detector pairs are shown as lines
 # between the optodes, channels (the mid point of source-detector pairs) are
-# shown as dots.
+# shown as orange dots. Source are optionally shown as red dots and detectors
```

Comment:
Channels are also not shown by default / are optional.

Reply:
added